src/components/Messages/MessageList.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+
+import * as React from "react";
+// TODO import { FlatList, Pressable, Text, Image } from "react-native";
+import { FlatList, Text } from "react-native";
+// TODO import { useNavigation } from "@react-navigation/native";
+
+// TODO import { imageStyles, viewStyles, textStyles } from "../../styles/projects/projects";
+import { textStyles } from "../../styles/projects/projects";  // TODO need to style
+
+type Props = {
+  loading: boolean,
+  messageList: Array<Object>,
+  testID: string
+}
+
+const MessageList = ( {
+  loading,
+  messageList,
+  testID
+}: Props ): React.Node => {
+  // TODO const navigation = useNavigation( );
+
+  const renderMessages = ( { item } ) => {
+    // TODO const navToProjectDetails = ( ) => navigation.navigate( "ProjectDetails", { id: item.id } );
+    return (
+      // TODO <Pressable
+      //   onPress={navToProjectDetails}
+      //   style={viewStyles.row}
+      //   testID={`Project.${item.id}`}
+      // >
+        <Text style={textStyles.projectName}>{item.subject}</Text>
+      // TODO </Pressable>
+    );
+  };
+
+  return (
+    <FlatList
+      data={messageList}
+      renderItem={renderMessages}
+      testID="Message.list" //Q: Is this for automated testing?  How should I use?
```

Comment:
Yes, `testID` is for automated testing. We've been using jest and setting up unit tests to check that some of the basic app functionality is working -- making sure data is rendered correctly on the page, that button taps do what we expect them to, and that a user can navigate away from their current screen.

Since you're already looking at ProjectDetails, here's a link to the ProjectDetails tests where you can see how we're setting up the unit tests.

https://github.com/inaturalist/iNaturalistReactNative/blob/03392f7da3a35952149488a12d4b1d323123d260/tests/unit/components/Projects/ProjectDetails.test.js